tests/testIngestCatalog.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+
+#
+# LSST Data Management System
+#
+# Copyright 2008-2015  AURA/LSST.
```

Comment:
This is not brand new code - it comes from testIngestSource.py in datarel, which has the following line in it:
```
# Copyright 2008, 2009, 2010 LSST Corporation.
```